### ISSUE TITLE
[Bug]: sp-divider not visible in Windows High Contrast mode

### ISSUE THREAD

**Issue by jnurthen** [adobe/spectrum-web-components#3277]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

sp-divider

### Expected behavior

Should be visible in High Contrast mode

### Actual behavior

Currently not visible in High Contrast mode

### How can we reproduce this issue?

1. Go to https://opensource.adobe.com/spectrum-web-components/components/divider/
2. Turn on high Contrast mode (or emulation)
3. Note that the dividers disappear

### Sample code that illustrates the problem

Note - compare to spectrum-css dividers work fine in high contrast mode (https://opensource.adobe.com/spectrum-css/divider.html)

**Comment by najikahalsema** [comment]
It's interesting that you see the sp-divider in high contrast mode on the Spectrum CSS website. It doesn't seem like any CSS rules for high contrast exist in the `spectrum-divider` css. I wonder why this and the other issues we have with WHCM happen. Doesn't seem like we always have rules for WHCM. Looking through the code base on main, looks like only Coachmark, ClearButton, and StatusLight. 

I don't know enough about WHCM or how the CSS gets applied, so I will need to investigate more. I'm just putting these notes here for myself and whomever else may benefit.

**Comment by pfulton** [comment]
We recently landed a [change](https://github.com/adobe/spectrum-css/commit/c37b5f7e9dbca9a6078cee1586ee180034bbc3b0) in Spectrum CSS that should have corrected this issue. Updating to the latest version (`2.1.6`) should get those new styles to you.

Please let us know if that's not the case!

**Comment by najikahalsema** [comment]
Confirming that I just updated divider and was able to see the divider in WHCM. I used assistiv labs to emulate.
